addons/account/models/partner.py

```diff
@@ -538,6 +538,7 @@ def _get_company_currency(self):
     duplicated_bank_account_partners_count = fields.Integer(
         compute='_compute_duplicated_bank_account_partners_count',
     )
+    is_coa_installed = fields.Boolean(store=False, default=lambda partner: bool(partner.env.company.chart_template_id))
```

Comment:
Note that `partner` in the lambda is misleading because it will always be an empty recordset.
I usually prefer `self` because I would less expect it to be the record being created in that case :shrug: 

Just a small detail, no need to change :shrug: